manifests/base.pp

```diff
@@ -32,11 +32,13 @@
   # and it needs the ruby-dev package installed to build
   #include ruby::devel
 
-  case $::operatingsystemmajrelease {
-    '7':    { $couchrest_version = '1.2'}
-    default:{ $couchrest_version = 'latest'}
+  if versioncmp($operatingsystemrelease, '8') < 0 {
```

Comment:
could you make $operatingsystemrelease be $::operatingsystemrelease to qualify the variable?